InMemoryRepositories/ShiftSwitchRequestTimeframeInMemoryRepository.cs

```diff
@@ -0,0 +1,62 @@
+﻿using Entities;
+using RepositoryContracts;
+using System.Linq;
+
+namespace InMemoryRepositories;
+
+public class ShiftSwitchRequestTimeframeInMemoryRepository : IShiftSwitchRequestTimeframeRepository
+{
+
+    private readonly List<ShiftSwitchRequestTimeframe> timeframes = new List<ShiftSwitchRequestTimeframe>();
+
+    public async Task<ShiftSwitchRequestTimeframe> AddAsync(ShiftSwitchRequestTimeframe timeframe)
+    {
+        if (timeframe.Id == 0)
+        {
+            timeframe.Id = timeframes.Any() ? timeframes.Max(t => t.Id) + 1 : 1;
+        }
```

Comment:
Shift switching is yet to be implemented but just got a question why timeframes have id? or is it id of a request?

Reply:
every entity has its own id assigned in java part and request has list of timeframes, thats how they are put together